google_api_lib/tasks.py

```diff
@@ -620,51 +620,60 @@ def update_active_users(self, hunt_id):
         else:
             body["pageToken"] = response["nextPageToken"]
 
-    with transaction.atomic():
-        user_puzzle_pairs = [
-            f"{user_pk}-{puzzle_pk}" for (user_pk, puzzle_pk) in latest.keys()
-        ]
-        old_activities = (
-            PuzzleActivity.objects.annotate(
-                user_puzzle_pair=Concat(
-                    F("user_id"), Value("-"), F("puzzle_id"), output_field=CharField()
+    try:
+        with transaction.atomic():
+            user_puzzle_pairs = [
+                f"{user_pk}-{puzzle_pk}" for (user_pk, puzzle_pk) in latest.keys()
+            ]
+            old_activities = (
+                PuzzleActivity.objects.annotate(
```

Comment:
Do we have a db index for user/puzzle?

Reply:
No, should we?

Reply:
i feel like our tables are small enough that it's not worth it but could be wrong